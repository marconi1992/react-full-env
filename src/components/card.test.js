import React from 'react'
import renderer from 'react-test-renderer'
import Card from './card'

describe('<Card />', () => {
  const props = {
    title: 'Test Title'
  }
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Card {...props} />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})