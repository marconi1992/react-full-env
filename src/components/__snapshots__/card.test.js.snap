// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders correctly 1`] = `
<div
  className="card"
>
  <h3>
    Test Title
  </h3>
</div>
`;
